checker/manual/external-tools.tex

```diff
@@ -324,60 +324,90 @@ \section{Maven\label{maven}}
             <artifactId>checker-qual</artifactId>
             <version>\ReleaseVersion{}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>checker</artifactId>
+            <version>\ReleaseVersion{}</version>
+        </dependency>
+        <!-- The annotated JDK to use (change to jdk7 if using Java 7) -->
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>jdk8</artifactId>
+            <version>\ReleaseVersion{}</version>
+        </dependency>
     </dependencies>
 \end{alltt}
 
-\item Direct the Maven compiler plugin to use the \code{javac\_maven} script.
-Change the reference to the \code{maven-compiler-plugin} within the \code{<plugins>}
-section, or add it if it is not present.
+\item Use a Maven property to hold the location of the
+  annotated JDK (which was declared as a Maven dependency above).
+To set the value of this property automaticaly, use the Maven Dependency plugin.
+
+First, create the property in the \code{properties} section of the POM:
 
 \begin{alltt}
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-          <fork>true</fork>
-          <executable>${env.CHECKERFRAMEWORK}/checker/bin/javac_maven</executable>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+<properties>
+    <!-- This property will be set by the Maven Dependency plugin -->
+    <annotatedJdk>${org.checkerframework:jdk8:jar}</annotatedJdk>
```

Comment:
To support working with projects such as Guava, add another property called jsr308Javac that points to org.checkerframework:compiler:jar